### PR TITLE
8133 studio should have render settings exposed

### PR DESCRIPTION
**Pull request by SYBIOTE**
## Summary

added fully fleshed checkboxes

## References

closes #_insert number here_

## Checklist
- [ ] If this PR is still a WIP, convert to a draft
- [ ] When this PR is ready, mark it as "Ready for review"
- [ ] [ensure all checks pass](https://github.com/etherealengine/etherealengine/wiki/Testing-&-Contributing)
- [ ] Changes have been manually QA'd
- [ ] Changes reviewed by at least 2 approved reviewer

## QA Steps

_List any additional steps required to QA the changes of this PR, as well as any supplemental images or videos._

### CODE REVIEW

**Review by HexaField — CHANGES_REQUESTED** [review]
Screenshot from 2023-07-12 15-22-42

This is what was meant by render settings, not the scene render settings. We should buff this out into a modal instead of a dropdown, and include the post processing button.

We then need to add conditions in all the places where the render settings affect the renderer, to use these new editor settings instead of user profile settings when in the editor.

**Comment by SYBIOTE** [comment]
Understood, I'm on it, do we also want the settings to disappear from the user config when in editor mode?
And what should we have in the modal?
Use shadows 
Use post processing
The render type dropdown 
What else?

File: packages/editor/src/components/properties/RenderSettingsEditor.tsx [hunk 1]
```diff
@@ -107,19 +110,37 @@ const ShadowTypeOptions = [
...
+  const rendererSettingsState = useComponent(props.entity, RenderSettingsComponent)
+  const rendererState = useHookstate(getMutableState(RendererState))
 
   return (
     <PropertyGroup
       name={t('editor:properties.renderSettings.name')}
       description={t('editor:properties.renderSettings.description')}
     >
+      <InputGroup
+        name="Use Post Processing"
```
Comment (hanzlamateen) [inline packages/editor/src/components/properties/RenderSettingsEditor.tsx:122]: Not sure if its a convention or so. But I think spaces & uppercaps are not recommended in `name` and `id` attributes:

https://courses.cs.washington.edu/courses/cse154/17au/styleguide/html-css/naming-conventions-html.html

Comment (SYBIOTE) [reply packages/editor/src/components/properties/RenderSettingsEditor.tsx:122]: Yeah I agree and I followed the code to maintain uniformity

**Comment by HexaField** [comment]
> Understood, I'm on it, do we also want the settings to disappear from the user config when in editor mode? And what should we have in the modal? Use shadows Use post processing The render type dropdown What else?

- `Use Shadows` corresponds to the render type, so we just need render type
- post processing
- shadow resolution from this PR https://github.com/EtherealEngine/etherealengine/pull/8267

**Comment by SYBIOTE** [comment]
so a checkbox, for use post processing, 
under it the resolution
under it the drop down 


as far as i know, the resolution is for overall graphics 
so i wont tie it to a condition ( selected shadows ) and keep it as the second in the modal

**Pull request merged**